news/add-lzf-test.rst

```diff
@@ -0,0 +1,9 @@
+Building h5py
+-------------
+
+* Build libhdf5 1.12.x in production mode
```

Comment:
I'm going to get a bit pickier about news entries after the release, having assembled them into release notes for 3.0. This kind of thing makes sense as a commit message, but release notes should be written addressing someone upgrading from a previous release: what has changed that they might care about? This change is important, but I think we're basically just getting back to the state we were in for previous releases (when wheels were built separately), so it probably doesn't need a news entry.

The 'Building h5py' section is primarily for notes that people might need if they build h5py from source, rather than details of how we build it for the wheels or the tests.

Reply:
Yup, I think we can drop the news entry for this - thanks. I'll make a PR to expand the documentation on news entries.

Reply:
news entry dropped